src/utils/DateUtils.ts

```diff
@@ -437,4 +423,48 @@ export class DateUtils {
       ('0' + (((moment(to).valueOf() - moment(from).valueOf()) % (1000 * 60)) / 1000).toFixed()).slice(-2)
     );
   }
+
+  static get currentGlobalizeCalendar(): GlobalizeCalendar {
+    return Globalize.culture(DateUtils.currentLocale).calendar as GlobalizeCalendar;
+  }
+
+  static get currentLocale(): string {
```

Comment:
Since we are transforming the locale value for momentjs, I think it would be good to mention that in the name. Maybe:
`currentLocaleCompatibleWithMomentjs` or `momentjsCompatibleLocale`